index.js

```diff
@@ -65,4 +66,17 @@ function deleteTask(event) {
   let newTasksList = tasksList.filter((task) => task.id !== id);
   tasksList = newTasksList;
   renderTask(tasksList);
-}
\ No newline at end of file
+}
+
+function toggleTask(event) {
+  let li = event.target.parentNode.parentNode;
+  let newTasksList = tasksList.map((task) => {
+    if (task.id === li.id) {
+      return { id: li.id, text: task.text, completed: !task.completed };
```

Comment:
it could be written also like `(task => task.id !== li.id ? task : {...task, completed: !task.completed})`. 
This way you can update a single item in an array. 
I recommend to remember this pattern.